From 3147ae2a63f10f9bbdd0a617b450ff8b9868e60f Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 20 Sep 2019 17:51:09 +0800
Subject: [PATCH] compat fip with openssl 1.0.2

In /usr/lib64/ssl/fips-2.0/include/openssl/opensslv.h
...
define OPENSSL_VERSION_NUMBER  0x10100000L
...
Since fips include file compat with openssl 1.1.0, do not include it
in Yocto

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 src/filehmac.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/filehmac.c b/src/filehmac.c
index a8eef00..0b36cec 100644
--- a/src/filehmac.c
+++ b/src/filehmac.c
@@ -41,7 +41,6 @@
 #include <sys/wait.h>
 
 #if defined(WITH_OPENSSL)
-#include <openssl/fips.h>
 #include <openssl/evp.h>
 #include <openssl/hmac.h>
 #elif defined(WITH_NSS)
-- 
2.7.4

